src/main/java/org/fundacionjala/sfdc/pageobjects/accounts/SFANewModifyPage.java

```diff
@@ -76,65 +60,85 @@ public void setParentAccountTextField(final String parentAccount) {
 
     /**
      * Method to set accountNumber attribute.
+     *
      * @param accountNumber the value.
      */
     public void setAccountNumberTextField(final String accountNumber) {
+        this.accountNumberTextField.clear();
         this.accountNumberTextField.sendKeys(accountNumber);
     }
 
     /**
      * Method to set accountSite attribute.
+     *
      * @param accountSite the value.
      */
     public void setAccountSiteTextField(final String accountSite) {
+        this.accountSiteTextField.clear();
         this.accountSiteTextField.sendKeys(accountSite);
     }
 
     /**
      * Method to set type attribute.
+     *
      * @param type the value.
      */
     public void setAccountTypeList(final String type) {
+        //this.typeList.clear();
         this.typeList.sendKeys(type);
     }
 
     /**
      * Method to set industry attribute.
+     *
      * @param industry the value.
      */
     public void setIndustryList(final String industry) {
+        //this.industryList.clear();
         this.industryList.sendKeys(industry);
     }
 
     /**
      * Method to set annualRevenue attribute.
+     *
      * @param annualRevenue the value.
      */
     public void setAnnualRevenueTextField(final String annualRevenue) {
+        this.annualRevenueTextField.clear();
         this.annualRevenueTextField.sendKeys(annualRevenue);
     }
 
 
-
     /**
-     * Method to click button.
+     * @param values account fields.
+     * @return map of account fields.
      */
-    public void clickSaveNewAccountButton() {
-        getSaveNewAccountButton().click();
+    public Map<Inputs, Value> getStrategyStepMap(final Map<Inputs, String> values) {
+        Map<Inputs, Value> strategyMap = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Map to an EnumMap. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1640)